addons/advanced_fatigue/functions/fnc_calculateBodyHeatStress.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+* Author: Pasrules
+* Calculates body heat stress
+*
+* Arguments:
+* 0: bodyTemperature <NUMBER>
+*
+* Return Value:
+* heatStress <NUMBER>
+*
+* Example
+* [37] call ace_advanced_fatigue_fnc_calculateBodyHeatStress
+*/
+
+#include "script_component.hpp"
+params ["_bodyTemp"];
+
+private _male = [87,177,37];
+private _normalTemp = _male select 2;
```

Comment:
I quite like the Array you proposed, I'm going to rebuild this to be one function